cfme/tests/test_manual.py

```diff
@@ -20162,103 +18126,33 @@ def test_user_cloud_memory_quota_by_services():
     Polarion:
         assignee: ghubale
         casecomponent: Configuration
-        caseimportance: low
-        initialEstimate: 1/2h
-        startsin: 5.8
-    """
-    pass
-
-
-@pytest.mark.manual
-@test_requirements.c_and_u
-@pytest.mark.tier(3)
-def test_crosshair_op_datastore_vsphere6():
-    """
-    Requires:
-    C&U enabled Vsphere-6 appliance.
-    Steps:
-    1. Navigate to Datastores [Compute > infrastructure>Datastores]
-    2. Select any available datastore
-    3. Go for utilization graphs [Monitoring > Utilization]
-    4. Check data point on graphs ["Used Disk Space", "Hosts", "VMs"]
-    using drilling operation on the data points
-    5.  check "chart" and "display" options working properly or not
-
-    Polarion:
-        assignee: nachandr
-        casecomponent: CandU
-        caseimportance: medium
-        initialEstimate: 1/12h
-    """
-    pass
-
-
-@pytest.mark.manual
-@test_requirements.reconfigure
-@pytest.mark.tier(1)
-def test_vm_reconfig_add_remove_disk_hot_vsphere67_nested_independent_persistent_thick():
-    """
-    Polarion:
-        assignee: nansari
-        casecomponent: Infra
-        initialEstimate: 1/3h
-        startsin: 5.7
-    """
-    pass
-
-
-@pytest.mark.manual
-@test_requirements.reconfigure
-@pytest.mark.tier(1)
-def test_vm_reconfig_add_remove_disk_hot_vsphere67_nested_independent_nonpersistent_thin():
-    """
-    Polarion:
-        assignee: nansari
-        casecomponent: Infra
-        initialEstimate: 1/3h
-        startsin: 5.7
-    """
-    pass
-
-
-@pytest.mark.manual
-@test_requirements.reconfigure
-@pytest.mark.tier(1)
-def test_vm_reconfig_add_remove_disk_hot_vsphere67_nested_independent_persistent_thin():
-    """
-    Polarion:
-        assignee: nansari
-        casecomponent: Infra
-        initialEstimate: 1/3h
-        startsin: 5.7
-    """
-    pass
-
-
-@pytest.mark.manual
-@test_requirements.reconfigure
-@pytest.mark.tier(1)
-def test_vm_reconfig_add_remove_disk_hot_vsphere67_nested_persistent_thin():
-    """
-    Polarion:
-        assignee: nansari
-        casecomponent: Infra
-        initialEstimate: 1/3h
-        startsin: 5.7
+        caseimportance: low
+        initialEstimate: 1/2h
+        startsin: 5.8
     """
     pass
 
 
 @pytest.mark.manual
-@test_requirements.reconfigure
-@pytest.mark.tier(1)
-def test_vm_reconfig_add_remove_disk_hot_vsphere67_nested_persistent_thick():
+@test_requirements.c_and_u
+@pytest.mark.tier(3)
+def test_crosshair_op_datastore_vsphere6():
     """
+    Requires:
+    C&U enabled Vsphere-6 appliance.
+    Steps:
+    1. Navigate to Datastores [Compute > infrastructure>Datastores]
+    2. Select any available datastore
+    3. Go for utilization graphs [Monitoring > Utilization]
+    4. Check data point on graphs ["Used Disk Space", "Hosts", "VMs"]
+    using drilling operation on the data points
+    5.  check "chart" and "display" options working properly or not
```

Comment:
this should be in `testSteps`

Reply:
This is not my FA, I have not added anything there and actually I have deleted them